frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -626,6 +850,16 @@ export const ClusterSettingsPage: React.SFC<ClusterSettingsPageProps> = ({ match
   const resources = [
     { kind: clusterVersionReference, name: 'version', isList: false, prop: 'obj' },
     { kind: clusterAutoscalerReference, isList: true, prop: 'autoscalers', optional: true },
+    {
+      kind: referenceForModel(MachineConfigPoolModel),
```

Comment:
We probably want to handle cases where the user doesn't have authority to watch MCPs or the one of the expected MCPs is not there. (For instance, you can have a 3 node cluster without workers.)

Make this `optional` and hide the progress bar if the MCP is missing or has 0 replicas?

Reply:
I think I've addressed this.  PTAL, @spadgett.